app/assets/javascripts/components/story/StoryNotes.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Note from 'components/notes/Note';
+import NoteForm from 'components/notes/NoteForm'
+
+class StoryNotes extends React.Component {
+  constructor() {
+    super();
+
+    this.renderNotes = this.renderNotes.bind(this);
```

Comment:
You don't need those `binds` here since you're not passing the method as a callback to other function. So you won't even need the constructor, you can simply remove it.